commit b6eeb7a1f7efa591504070b606be655e27e6e9c2
Author: Chris Wilson <chris@chris-wilson.co.uk>
Date:   Wed Nov 5 13:03:41 2014 +0000

    Disable DRI3 by default
    
    The external libraries, both in git, and especially shipping already
    enabled in distributions, are buggy and lead to server crashes and
    lockups. Caveat emptor.
    
    Signed-off-by: Chris Wilson <chris@chris-wilson.co.uk>

diff --git a/configure.ac b/configure.ac
index 328b4e9..cbfc472 100644
--- a/configure.ac
+++ b/configure.ac
@@ -339,10 +339,10 @@ AC_ARG_ENABLE(dri2,
 	      [DRI2=$enableval],
 	      [DRI2=yes])
 AC_ARG_ENABLE(dri3,
-	      AS_HELP_STRING([--disable-dri3],
-			     [Disable DRI3 support [[default=yes]]]),
+	      AS_HELP_STRING([--enable-dri3],
+			     [Enable DRI3 support [[default=no]]]),
 	      [DRI3=$enableval],
-	      [DRI3=yes])
+	      [DRI3=no])
 
 AC_ARG_ENABLE(xvmc, AS_HELP_STRING([--disable-xvmc],
                                   [Disable XvMC support [[default=yes]]]),
